console/src/main/scripts/plugins/metrics/html/directives/alert-summary.html

```diff
@@ -23,7 +23,7 @@
     <strong>Non Heap Memory</strong>: The Non Heap memory usage was {{alert.avg / 1024 / 1024 | number:2}} MB.
   </div>
   <div ng-switch-when="GARBA">
-    <strong>Garbage collection</strong>: The GC averaged more than {{alert.avg | number:2}} ms per minute.
+    <strong>Garbage collection</strong>: The GC averaged more than {{alert.avg | number:0}} ms per minute.
```

Comment:
That was actually the target of the JIRA for this commit, but in any case, I created a new one: HAWKULAR-946